app/src/main/java/edu/uc/zhukv/droneradarmap/MapsActivity.kt

```diff
@@ -20,7 +20,9 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
 
     private lateinit var mMap: GoogleMap
 
-    private val OWM_TILE_URL = "http://tile.openweathermap.org/map/%s/%d/%d/%d.png?appid=d6d46d84c231bd013c9f0088629b0eb8"
+    //Renamed this without underscore as per naming conventions insisted by Kotlin Formatting
```

Comment:
Updated the naming to something more meaningful and simple. Might want to consider just making this a public constant instead.